shell/common/api/atom_api_shell.cc

```diff
@@ -71,6 +71,13 @@ v8::Local<v8::Promise> OpenExternal(const GURL& url, mate::Arguments* args) {
   return handle;
 }
 
+bool MoveItemToTrash(const base::FilePath& full_path, mate::Arguments* args) {
```

Comment:
`bool MoveItemToTrash(mate::Arguments* args)` , try to reuse args and perform the required processing